models/clm/clm_to_dart.rst

```diff
@@ -0,0 +1,198 @@
+PROGRAM ``clm_to_dart``
+=======================
+
+Overview
+--------
+
+``clm_to_dart`` is used to replace indeterminate values in the CLM restart file
+with the *_FillValue* value specified by the variable attribute. This is necessary
```

Comment:
Is 'indeterminate' the same as 'missing_value'?

Reply:
not really ...   in the CLM files, the 'missing_value' and the '_FillValue' have the same value. What I'm calling indeterminate are values that are explicitly _not_ '_FillValue' - but should be.